Meta/gn/secondary/Kernel/BUILD.gn

```diff
@@ -792,7 +792,6 @@ executable("Kernel_bin") {
     "Syscalls/utimensat.cpp",
     "Syscalls/waitid.cpp",
     "Syscalls/write.cpp",
-    "TTY/ConsoleManagement.cpp",
     "TTY/MasterPTY.cpp",
     "TTY/PTYMultiplexer.cpp",
```

Comment:
Just saying that the `BUILD.gn` for the kernel sources is utterly out of date. Maybe a thing for a future PR, but not here :)